cpp/Brewfile

```diff
@@ -28,8 +28,6 @@ brew "git"
 brew "glog"
 brew "googletest"
 brew "grpc"
-brew "llvm"
-brew "llvm@12"
```

Comment:
Can we add `llvm@14` instead?

```ruby
brew "llvm@14"
```

I assume that adding `llvm@14` doesn't cause `llvm@14` (re)install on GitHub Actions because `llvm@14` is already installed.

Reply:
done